ocrd/ocrd/cli/workspace.py

```diff
@@ -387,6 +387,21 @@ def workspace_remove_file(ctx, id, force, keep_file):  # pylint: disable=redefin
     workspace.save_mets()
 
 
+# ----------------------------------------------------------------------
+# ocrd workspace rename-group
+# ----------------------------------------------------------------------
+
+@workspace_cli.command('rename-group')
+@click.argument('OLD', nargs=1)
+@click.argument('NEW', nargs=1)
+@pass_workspace
+def rename_group(ctx, old, new):
+    """
+    Rename fileGrp (USE attribute ``NEW`` to ``OLD``).
+    """
+    workspace = Workspace(ctx.resolver, directory=ctx.directory, mets_basename=basename(ctx.mets_url))
+    workspace.rename_file_group(old, new)
+
```

Comment:
I think there is a `workspace.save_mets()` missing here

Reply:
Yep, that change fixes the issue. @kba Do you add it? I think I cannot push to the PR

Reply:
~~~
diff --git a/ocrd/ocrd/cli/workspace.py b/ocrd/ocrd/cli/workspace.py
index ccc199f4..a0defb4e 100644
--- a/ocrd/ocrd/cli/workspace.py
+++ b/ocrd/ocrd/cli/workspace.py
@@ -401,6 +401,7 @@ def rename_group(ctx, old, new):
     """
     workspace = Workspace(ctx.resolver, directory=ctx.directory, mets_basename=basename(ctx.mets_url))
     workspace.rename_file_group(old, new)
+    workspace.save_mets()
 
 # ----------------------------------------------------------------------
 # ocrd workspace remove-group
~~~

Reply:
I'll add a fix, thanks for spotting this.

Reply:
485d284d